### ISSUE TITLE
init-container error: unable to install the chart

### ISSUE THREAD

**Issue by joancafom** [jenkinsci/helm-charts#519]
**Describe the bug**

The latest versions of the Chart do not seem to be able to be installed. The init-container of the main pod does produce an error on start-up, and leads to an `Init:CrashLoopBackOff` state. 

The associated error logged by the init-container is the following:

```bash
$ kubectl logs jenkins-0 -c init
...
Will install new plugin workflow-api 1106.v6a04bac22f47
io.jenkins.tools.pluginmanager.impl.AggregatePluginPrerequisitesNotMetException: Multiple plugin prerequisites not met:
Plugin kubernetes:1.30.11 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54,
Plugin workflow-aggregator:2.6 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54,
Plugin git:4.10.0 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54
	at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:222)
	at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:171)
	at io.jenkins.tools.pluginmanager.cli.Main.main(Main.java:70)
	Suppressed: io.jenkins.tools.pluginmanager.impl.PluginDependencyException: Plugin kubernetes:1.30.11 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54
		at io.jenkins.tools.pluginmanager.impl.PluginManager.resolveRecursiveDependencies(PluginManager.java:1074)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.findPluginsAndDependencies(PluginManager.java:649)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:214)
		... 2 more
	Suppressed: io.jenkins.tools.pluginmanager.impl.PluginDependencyException: Plugin workflow-aggregator:2.6 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54
		at io.jenkins.tools.pluginmanager.impl.PluginManager.resolveRecursiveDependencies(PluginManager.java:1074)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.findPluginsAndDependencies(PluginManager.java:649)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:214)
		... 2 more
	Suppressed: io.jenkins.tools.pluginmanager.impl.PluginDependencyException: Plugin git:4.10.0 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54
		at io.jenkins.tools.pluginmanager.impl.PluginManager.resolveRecursiveDependencies(PluginManager.java:1074)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.findPluginsAndDependencies(PluginManager.java:649)
		at io.jenkins.tools.pluginmanager.impl.PluginManager.start(PluginManager.java:214)
		... 2 more
Multiple plugin prerequisites not met:
Plugin kubernetes:1.30.11 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54,
Plugin workflow-aggregator:2.6 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54,
Plugin git:4.10.0 (via credentials:1055.v1346ba467ba1) depends on configuration-as-code:1.55, but there is an older version defined on the top level - configuration-as-code:1.54
```

**Version of Helm and Kubernetes**:

Helm Version:

```console
$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

Kubernetes Version:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:20:00Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Which version of the chart**:

`3.9.4` and `3.9.3`

**What happened**:

The init-container produces an error and pods never reach the`READY` status.

**What you expected to happen**:

To have the application working

**How to reproduce it** (as minimally and precisely as possible):

Simply try to install the latest version of the chart:

```bash
$ helm install jenkins jenkins/jenkins --version 3.9.4
NAME: jenkins
LAST DEPLOYED: Thu Dec 16 18:02:17 2021
NAMESPACE: default
STATUS: deployed
REVISION: 1
NOTES:
1. Get your 'admin' user password by running:
  kubectl exec --namespace default -it svc/jenkins -c jenkins -- /bin/cat /run/secrets/chart-admin-password && echo
2. Get the Jenkins URL to visit by running these commands in the same shell:
  echo http://127.0.0.1:8080
  kubectl --namespace default port-forward svc/jenkins 8080:8080

3. Login with the password from step 1 and the username: admin
4. Configure security realm and authorization strategy
5. Use Jenkins Configuration as Code by specifying configScripts in your values.yaml file, see documentation: http:///configuration-as-code and examples: https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos

For more information on running Jenkins on Kubernetes, visit:
https://cloud.google.com/solutions/jenkins-on-container-engine

For more information about Jenkins Configuration as Code, visit:
https://jenkins.io/projects/jcasc/


NOTE: Consider using a custom image with pre-installed plugins

$ kubectl get pods
NAME        READY   STATUS       RESTARTS   AGE
jenkins-0   0/2     Init:Error   2          36s
```

**Anything else we need to know**: NA

**Comment by nguyencatpham** [comment]
i meet the same problem

**Comment by torstenwalter** [comment]
It's fixed with #525
